spec/core/query_spec.rb

```diff
@@ -240,9 +240,9 @@
     context Daru::DataFrame do
       before do
         @df = Daru::DataFrame.new({
-          number: [1,2,3,4,5,6],
-          sym: [:one, :two, :three, :four, :five, :six],
-          names: ['sameer', 'john', 'james', 'omisha', 'priyanka', 'shravan']
+          number: [1,2,3,4,5,6,nil],
+          sym: [:one, :two, :three, :four, :five, :six, :seven],
+          names: ['sameer', 'john', 'james', 'omisha', 'priyanka', 'shravan', Float::NAN.to_s]
```

Comment:
That was meant to be `Float::NAN` missing type as mentioned [here](https://github.com/SciRuby/daru/blob/master/lib/daru.rb#L33). I will change that in the next commit, along with other changes. :smile: